homeassistant/components/neevo/__init__.py

```diff
@@ -0,0 +1,130 @@
+"""The Nee-Vo Tank Monitoring integration."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from aiohttp.client_exceptions import ClientError
+from pyneevo import NeeVoApiInterface
+from pyneevo.errors import (
+    GenericHTTPError,
+    InvalidCredentialsError,
+    InvalidResponseFormat,
+    PyNeeVoError,
+)
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, Platform
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import COORDINATOR, DOMAIN, TANKS
+
+_LOGGER = logging.getLogger(__name__)
+
+PUSH_UPDATE = "neevo.push_update"
+INTERVAL = 60
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
```

Comment:
If those variables are not used outside this module, user `_` prefix to mark them as private: https://docs.python.org/3/tutorial/classes.html#private-variables